api/src/wfl/executor.clj

```diff
@@ -225,18 +225,49 @@
                     {:method_configuration_version inc'd}
                     ["id = ?" id]))))
 
+(defn ^:private submission-created-slack-msg
+  "Return a mrkdwn Slack message to be emitted for newly created `submission`."
+  [{:keys [workspace]    :as _executor}
+   {:keys [submissionId] :as _submission}
+   snapshot]
+  (let [emoji           (cromwell/emoji "Submitted")
+        submission-link (-> submissionId
+                            (firecloud/submission-url workspace)
+                            (slack/link submissionId))
+        snapshot-link   (-> snapshot
+                            datarepo/snapshot-url
+                            (slack/link (:name snapshot)))]
+    (str/join \newline
+              [(format "*%s Launched submission %s*" emoji submission-link)
+               (format "Snapshot %s" snapshot-link)])))
+
 (defn ^:private create-submission!
   "Update `executor` method configuration to use `reference`,
   then submit in `executor` workspace via Firecloud.
-  Return submission."
-  [{:keys [executor] :as workload} userComment reference]
-  (let [{:keys [workspace methodConfiguration]} executor]
-    (update-method-configuration! workload reference)
-    (log/debug "Initiating Terra submission"
-               :workload            (workloads/to-log workload)
-               :methodConfiguration methodConfiguration
-               :workspace           workspace)
-    (firecloud/submit-method workspace methodConfiguration userComment)))
+  Notify Slack watchers and return submission."
+  ([{:keys [executor] :as workload}
+    user-comment-note
+    reference
+    [_type snapshot   :as _source-object]]
```

Comment:
Heads up with some historical context:

We get this from the `source` queue, originally I thought this would be a snapshot but instead it's a pair:

```
[:datarepo/snapshot <snapshot>]
```

I think the original intention was to think of a future where we submitted something-that-isn't-a-snapshot (like a row in a Terra entity table).  But I elected not to fuss much with it.